common/trinary/benchmark/bench_trit_tryte.c

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2020 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+#include "common/trinary/trit_tryte.h"
+
+// The minimum and maximum input/output tryte size for perfomance testing
+#define MIN_TRYTE_SIZE 16
+#define MAX_TRYTE_SIZE 2048
```

Comment:
let's make it 2673 trytes, the length of a transaction.
```suggestion
#define MAX_TRYTE_SIZE 2673
```

Reply:
Done.